spec/models/student_spec.rb

```diff
@@ -2,4 +2,25 @@
 
 RSpec.describe Student, "validations" do
   it { is_expected.to validate_presence_of(:name) }
+  it { is_expected.to validate_presence_of(:grade) }
+  it { is_expected.to validate_presence_of(:teacher_id) }
+  it { is_expected.to validate_presence_of(:hair_color)}
```

Comment:
👍 👍 For some practice in the future, try testing other types of validations such as numbers, email addresses, etc.